src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -841,11 +841,29 @@ namespace winrt::TerminalApp::implementation
             renameTabMenuItem.Icon(renameTabSymbol);
         }
 
+        Controls::MenuFlyoutItem duplicateTabMenuItem;
+        {
+            // "Duplicate Tab"
+            Controls::FontIcon duplicateTabSymbol;
+            duplicateTabSymbol.FontFamily(Media::FontFamily{ L"Segoe MDL2 Assets" });
+            duplicateTabSymbol.Glyph(L"\xF5ED");

```

Comment:
For anyone curious, this is the icon:
![Segoe MDL2 Asset Icon](https://user-images.githubusercontent.com/11050425/110154862-a19fba00-7d99-11eb-8a53-ad3a64ef46da.png)

Chromium edge uses this:
![Edge Icon](https://user-images.githubusercontent.com/11050425/110155031-d7dd3980-7d99-11eb-8bbc-911ae181855c.png)

but that's not a Segoe MDL2 Asset.

I wish there was a better icon but I couldn't find one. Seems like Segoe MDL2 Assets don't include any tab-related icons.

Reply:
Yeah. Didn't find anything less confusing